cmd/plural/shell.go

```diff
@@ -77,5 +77,6 @@ func handleShellPurge(c *cli.Context) error {
 	}
 
 	client := api.NewClient()
-	return client.DeleteShell()
+	err := client.DeleteShell()
+	return api.GetErrorResponse(err, "DeleteShell")
```

Comment:
Same here. Should we not check if the error is nil first, just like in [pkg/crypto/age.go](https://github.com/pluralsh/plural-cli/pull/277/files#diff-42e3a757ec072a799849e121c4cd26da264e442216ec0369c1f7c477fdd983f9).